packages/lesswrong/server/emailComponents/NewPostEmail.tsx

```diff
@@ -131,9 +136,11 @@ const NewPostEmail = ({documentId, reason, hideRecommendations, classes}: {
       <hr />
     </div>}
     
-    {document.contents && <EmailContentItemBody className="post-body" dangerouslySetInnerHTML={{
-      __html: document.contents.html
-    }} />}
+    {document.contents && <ContentStyles contentType="post">
+      <EmailContentItemBody className="post-body" dangerouslySetInnerHTML={{
+        __html: document.contents.html
+      }} />
+    </ContentStyles>}
```

Comment:
I'm not sure whether it's the right call to use `<ContentStyles>` here. It adds a million billion styles, most of which are irrelevant for most posts, but we do want the styling to be broadly consistent with how posts look on the site so seems fine? I guess we could pull out a subset of the styles for emails, but I'm just trying to do a quick fix here so I'd prefer not to have to do that in this PR. I'm surprised there's no explicit styling here already - without this, the contents have very little styling:

![Screen Shot 2024-06-20 at 5 58 22 PM](https://github.com/ForumMagnum/ForumMagnum/assets/9057804/7e711122-abff-482c-ad89-623ab3a1cc64)

Reply:
I think making it exactly the same as the on-site styling is worth the cost 👍